src/black/schema.py

```diff
@@ -0,0 +1,19 @@
+import importlib.resources
+import json
+import sys
+from typing import Any
+
+
+def get_schema(tool_name: str = "cibuildwheel") -> Any:
+    "Get the stored complete schema for black's settings."
+    assert tool_name == "black", "Only black is supported."
```

Comment:
Then why is the default `cibuildwheel`?

Reply:
There’s a small, tiny chance I might have started from the file I wrote for cibuildwheel…